src/meta/meta_turtle.c

```diff
@@ -129,9 +129,7 @@ __metadata_load_bulk(WT_SESSION_IMPL *session)
         WT_ERR(cursor->get_value(cursor, &value));
         filecfg[1] = value;
         WT_ERR(__wt_direct_io_size_check(session, filecfg, "allocation_size", &allocsize));
-        WT_WITH_BUCKET_STORAGE(
-          NULL, session, ret = __wt_block_manager_create(session, key, allocsize));
-        WT_ERR(ret);
```

Comment:
OTOH, this is saying, use a NULL bucket storage, or just the default file system when creating this file.  It's a no-op, so can be removed.